docs/scoring.md

```diff
@@ -23,58 +23,56 @@ All audits in the SEO category are [equally weighted](https://github.com/GoogleC
 
 ## How is the accessibility score calculated?
 
-<!--
-  sum = auditRefs.reduce((sum, item) => sum += item.weight, 0);
-  auditRefs
-    .filter(a => a.weight)
-    .sort((a, b) => b.weight - a.weight)
-    .map(a => [undefined, a.id, `${(a.weight / sum * 100).toLocaleString(undefined, {maximumFractionDigits:1})}%`, undefined].join(' | '))
-    .join('\n')
-  -->
+<!-- node lighthouse-core/scripts/print-a11y-scoring.js -->
```

Comment:
I skipped over without parsing. What about something like this to make it slightly more obvious:

```suggestion
<!-- To regnerate score weights, run `node lighthouse-core/scripts/print-a11y-scoring.js`-->
```